source/server/listener_manager_impl.cc

```diff
@@ -338,7 +338,6 @@ bool ListenerManagerImpl::addOrUpdateListenerInternal(const envoy::api::v2::List
     ENVOY_LOG(debug, "duplicate/locked listener '{}'. no add/update", name);
     return false;
   }
-
```

Comment:
Please revert spurious newline removals here and elsewhere, it just makes the diffs larger for no reason